packages/braintree-integration/src/braintree-venmo/braintree-venmo-button-strategy.ts

```diff
@@ -200,10 +207,10 @@ export default class BraintreeVenmoButtonStrategy implements CheckoutButtonStrat
                         venmoButton.removeAttribute('disabled');
 
                         if (error) {
-                            return this._onError(error);
+                            return this.onError(error);
```

Comment:
In this case tokenize callback method will return `void`;

```suggestion
                            this.onError(error);
                            
                            return;
```

Reply:
fixed